tests/dragonfly/cluster_test.py

```diff
@@ -1433,8 +1433,7 @@ async def test_migration_with_key_ttl(df_factory):
     assert await nodes[1].client.execute_command("stick k_sticky") == 0
 
 
-@pytest.mark.skip("Flaky test")
-@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes", "migration_finalization_timeout_ms": 5})
```

Comment:
you change here the migration_finalization_timeout_ms default
in this test we check that when we cancel the migration due to disconnect we are restarting the migration again and we wait for the sync status
please explain how finalization timeout affect this and why?

Reply:
if we cancel the migration during finalization we should wait until the timeout is ended, and to prevent 30 seconds of waiting I have changed it to 5

Reply:
Why dont we break on waiting the finalization? because if the connection is closed so we should return with error while waiting for the migration finalization.

Reply:
we tried to reconnect until the timeout is happened